_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,412 @@
+
+```html
+<div class="col-container">
+	<div class="col">
+		<p>Interact with the box below this paragraph, for a surprise</p>
+	</div>
+
+	<div class="col">
+		<button onclick="alert('Surprise!')">Howdy</button>
+	</div>
+</div>
+```
+
+#### Passed Example 3
+
+The visual reference made by the word "right" is complemented by the non-visual reference made by the word "menu" to the content identified by the "Menu" heading.
+
+```html
+<head>
+	<title>Passed example 3 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<h1>Menu</h1>
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+			</ul>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 4
+
+This document is using the word "square" but in this case it is not describing web content.
+
+```html
+<p>A square is a regular quadrilateral with four equal sides and four right angles.</p>
+```
+
+#### Passed Example 5
+
+The following text is tilted and describes web content. But the [section of content](#section-of-content) also includes this word "this" which makes it apparent that the description is about the same content.
+
+```html
+<head>
+	<title>Passed example 5 9bd38c</title>
+	<style>
+		div.tilt {
+			height: 750px;
+			width: 150px;
+			-ms-transform: rotate(20deg); /* IE 9 */
+			-webkit-transform: rotate(20deg); /* Safari 3-8 */
+			transform: rotate(20deg);
+		}
+	</style>
+</head>
+<body>
+	<div class="tilt">look at this pieCe of tiLted text fOr clueS on whEre to find The monster.</div>
+</body>
+```
+
+#### Passed Example 6
+
+The button is indicated by the word "round". But the word is also included in the text of the element.
+
+```html
+<head>
+	<title>Passed example 6 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the round button, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Round button</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 7
+
+The images are indicated by the visual indicator words "narrow" and "wide". These words are also included in the accessible names of the images. Even through that indication is not visible, it is included in the accessibility tree which is sufficient to pass this rule.
+
+```html
+<p>The wide image is awesome. But the narrow image isn't.</p>
```

Comment:
These are not instructions (similar to the passed example 10), so the description should be updated to reflect that.

If you think they are part of instructions, then you need to review the expectation, because then instructions is ambiguous.